.gitignore

```diff
@@ -42,5 +42,11 @@ kls_database.db
 # IDE specific folders
 # IntelliJ
 .idea/
+# VS Code
 .vscode/
-.dir-local.el
\ No newline at end of file
+.dir-local.el
+# Eclipse
+.settings/
+.project
+# This one seems to be more widely adopted by many IDEs
+.classpath
```

Comment:
Should this ignore `my.classpath`? That is, the file generated by the ide-paths gradle task?

Reply:
Probably should ignore that as well. That was only meant to be a temporary file to help people set up an IDE to work better. Good catch